main/src/cgeo/geocaching/maps/mapsforge/v6/NewMap.java

```diff
@@ -663,7 +663,6 @@ private void switchTileLayer(final MapSource newSource) {
 
         // Exchange layer
         if (newLayer != null) {
-            this.mapView.getModel().displayModel.setFixedTileSize(newLayer.getFixedTileSize());
```

Comment:
Mentally this change doesn't align with the title for me.
Originally this line was required to get the switch between on- and offline mapsforge maps working (IIRC, it's been a while) without a crash.
I would be happy to better understand the background and effect.